geom.go

```diff
@@ -203,9 +205,13 @@ func (p Polygon) Clone() Polygon {
 type Op int
 
 const (
+	// UNION operation
```

Comment:
Those comments don't bring any value for me. I can't believe golint would require them this way. I don't think the standard library has them this way. Some first case I found in the stdlib: http://golang.org/pkg/unicode/#pkg-constants - the block with `UpperCase` etc. doesn't seem to have any comments. I prefer to have no useless comments here and have golint complain than add them to satisfy golint. On the other hand, a comment above `const (` line would be totally OK. If this would satisfy golint, please do it this way. Thanks.